GoogleForCreators/web-stories-wp
# PPC: Trigger checklist item calculations after initial element added

## Context
Part of work to make prepublish checklist less annoying.

## Summary
This PR works on not calculating any checklist items until the stories been interacted with. Some of the UI updates are taken care of in this PR here: https://github.com/google/web-stories-wp/pull/7277

Where this one is only focused on altering the actual calculation of checklist items.

## Relevant Technical Choices



## To-do



## User-facing changes
Checklist should now not be calculated until there's at least one element in the story.


## Testing Instructions
open a new story, see that there are no messages in the prepublish checklist. Add an element, and see that the messages now show up in the prepublish checklist.

open up an existing story with elements in it and see that right when it's open all the checklist items are present


### QA
Follow testing instructions above





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7138
